load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -213,14 +213,13 @@ func (s *relaysSuite) mapSessionInfoForLoadTestDurationFn(
 		// If the test duration is reached, stop sending requests
 		sendRelaysEndHeight := s.testStartHeight + s.relayLoadDurationBlocks
 		if blockHeight >= sendRelaysEndHeight {
+			testEndHeight := s.testStartHeight + s.testDurationBlocks*2
 
-			logger.Info().Msg("Stop sending relays, waiting for last claims and proofs to be submitted")
+			logger.Info().Msgf("Stop sending relays, waiting for last claims and proofs to be submitted: %d/%d", blockHeight-sendRelaysEndHeight, testEndHeight-sendRelaysEndHeight)
```

Comment:
1) This would be easier to read if these were extracted to variables.

```go
remainingRelayLoadBlocks := blockHeight - sendRelaysEndHeight
waitForSettlementBlocks := testEndHeight - sendRelaysEndHeight
```

2) I'm confused by what this is intended to communicate; perhaps using `logger.Info().Fields(...).Msg(...)` would further improve readability.